examples/nlp/glue_benchmark/glue_benchmark_with_bert.py

```diff
@@ -248,6 +246,16 @@
     pooler = SequenceClassifier(hidden_size=hidden_size, num_classes=num_labels, log_softmax=False)
     glue_loss = CrossEntropyLossNM()
 
+if args.bert_checkpoint is not None:
+    model.restore_from(args.bert_checkpoint)
+    logging.info(f"model restored from {args.bert_checkpoint}")
+
+    if args.task_head_checkpoint is not None:
```

Comment:
Isn't task_head_checkpoint param independent of bert_checkpoint? Shouldn't this be out of the block?